libflux/flux-core/src/ast/walk/mod.rs

```diff
@@ -266,7 +272,7 @@ where
     walk_rc(v, Rc::new(node));
 }
 
-pub fn walk_rc<'a, T>(v: &T, node: Rc<Node<'a>>)
+fn walk_rc<'a, T>(v: &T, node: Rc<Node<'a>>)
```

Comment:
Definitely need a second opinion here.

Reply:
I have often thought that we need to audit the public API for both our crates. I'm not sure it's safe to necessarily make these changes in a documentation patch (there are downstream consumers of this library directly). I'd prefer we just mark this `allow(missing_docs)` and leave the changes to the interface as they are.